diff -Naur davs2-1.6.old/source/common/vec/intrinsic_deblock_avx2.cc davs2-1.6/source/common/vec/intrinsic_deblock_avx2.cc
--- davs2-1.6.old/source/common/vec/intrinsic_deblock_avx2.cc	2019-06-08 09:11:36.496537065 +0200
+++ davs2-1.6/source/common/vec/intrinsic_deblock_avx2.cc	2019-06-08 09:12:32.051603322 +0200
@@ -30,15 +30,14 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
 
+#include "../common.h"
+#include "intrinsic.h"
 
 #if !HIGH_BIT_DEPTH
 __m128i c_0_128;
diff -Naur davs2-1.6.old/source/common/vec/intrinsic_idct_avx2.cc davs2-1.6/source/common/vec/intrinsic_idct_avx2.cc
--- davs2-1.6.old/source/common/vec/intrinsic_idct_avx2.cc	2019-06-08 09:11:36.496537065 +0200
+++ davs2-1.6/source/common/vec/intrinsic_idct_avx2.cc	2019-06-08 09:12:32.052603341 +0200
@@ -30,15 +30,15 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
 
+#include "../common.h"
+#include "intrinsic.h"
+
 /* disable warnings */
 #pragma warning(disable:4127)  // warning C4127: 条件表达式是常量
 
diff -Naur davs2-1.6.old/source/common/vec/intrinsic_inter_pred_avx2.cc davs2-1.6/source/common/vec/intrinsic_inter_pred_avx2.cc
--- davs2-1.6.old/source/common/vec/intrinsic_inter_pred_avx2.cc	2019-06-08 09:11:36.496537065 +0200
+++ davs2-1.6/source/common/vec/intrinsic_inter_pred_avx2.cc	2019-06-08 09:12:32.054603380 +0200
@@ -30,14 +30,15 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
+
+#include "../common.h"
+#include "intrinsic.h"
+
 #pragma warning(disable:4127)  // warning C4127: 条件表达式是常量
 
 #if !HIGH_BIT_DEPTH
diff -Naur davs2-1.6.old/source/common/vec/intrinsic_inter_pred.cc davs2-1.6/source/common/vec/intrinsic_inter_pred.cc
--- davs2-1.6.old/source/common/vec/intrinsic_inter_pred.cc	2019-06-08 09:11:36.496537065 +0200
+++ davs2-1.6/source/common/vec/intrinsic_inter_pred.cc	2019-06-08 09:12:32.054603380 +0200
@@ -30,15 +30,15 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
 
+#include "../common.h"
+#include "intrinsic.h"
+
 #if !HIGH_BIT_DEPTH
 /* ---------------------------------------------------------------------------
  */
diff -Naur davs2-1.6.old/source/common/vec/intrinsic_intra-pred_avx2.cc davs2-1.6/source/common/vec/intrinsic_intra-pred_avx2.cc
--- davs2-1.6.old/source/common/vec/intrinsic_intra-pred_avx2.cc	2019-06-08 09:11:36.497537084 +0200
+++ davs2-1.6/source/common/vec/intrinsic_intra-pred_avx2.cc	2019-06-08 09:12:32.055603399 +0200
@@ -30,15 +30,15 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
 
+#include "../common.h"
+#include "intrinsic.h"
+
 #if !HIGH_BIT_DEPTH
 
 void intra_pred_ver_avx(pel_t *src, pel_t *dst, int i_dst, int dir_mode, int bsx, int bsy)
diff -Naur davs2-1.6.old/source/common/vec/intrinsic_pixel_avx.cc davs2-1.6/source/common/vec/intrinsic_pixel_avx.cc
--- davs2-1.6.old/source/common/vec/intrinsic_pixel_avx.cc	2019-06-08 09:11:36.497537084 +0200
+++ davs2-1.6/source/common/vec/intrinsic_pixel_avx.cc	2019-06-08 09:12:32.055603399 +0200
@@ -30,15 +30,15 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
 
+#include "../common.h"
+#include "intrinsic.h"
+
 /* ---------------------------------------------------------------------------
  */
 void *davs2_memzero_aligned_c_avx(void *dst, size_t n)
diff -Naur davs2-1.6.old/source/common/vec/intrinsic_sao_avx2.cc davs2-1.6/source/common/vec/intrinsic_sao_avx2.cc
--- davs2-1.6.old/source/common/vec/intrinsic_sao_avx2.cc	2019-06-08 09:11:36.497537084 +0200
+++ davs2-1.6/source/common/vec/intrinsic_sao_avx2.cc	2019-06-08 09:12:32.055603399 +0200
@@ -30,15 +30,15 @@
  *    For more information, contact us at sswang @ pku.edu.cn.
  */
 
-#include "../common.h"
-#include "intrinsic.h"
-
 #include <mmintrin.h>
 #include <emmintrin.h>
 #include <tmmintrin.h>
 #include <smmintrin.h>
 #include <immintrin.h>
 
+#include "../common.h"
+#include "intrinsic.h"
+
 #if !HIGH_BIT_DEPTH
 #ifdef _MSC_VER
 #pragma warning(disable:4244)  // TODO: 修正编译warning
